### DISCUSSION TITLE
UL List id option

### DISCUSSION THREAD

**Discussion by TehGoodLivin** [PhilippeMarcMeyer/vanillaSelectBox#34] [Q&A] [answered]
First off, an awesome little plugin for a multi-selection list.

Just a simple idea to add support to the list. Adding the ID to the UL. I know I went through and added it to the js. Might be useful to add an option for adding an id to the list when you initialize.

```
if (self.userOptions.listid != "") {
    this.ul.id = self.userOptions.listid;
}
```

was about to write in part for a limit to selection (saw it in the discussion), looks like a few others had the same idea. 

Sample: [id.zip](https://github.com/PhilippeMarcMeyer/vanillaSelectBox/files/6118685/id.zip)

**Comment by PhilippeMarcMeyer** [comment]
Or the ul Id could be a derivative of the original select tag Id ?
originalId-ul
I would set an id in both the button part and the menu part (ul) that would
ease access in order to customise css after init

**Reply by TehGoodLivin** [reply]
Makes sense. Looking at adding event listeners for selection changes.

**Comment by PhilippeMarcMeyer** [comment, marked as answer]
There is a id in a parent DIV 
btn-group-{is of the original select tag}

example : 

```
<select id="test"></select>

<div class="vsb-main" id="btn-group-test">....

```
so you can get it with : 

```
document.querySelector("#btn-group-test ul");
```
Cheers
